support-services/src/main/scala/com/gu/aws/AwsCloudWatchMetricSetup.scala

```diff
@@ -25,15 +25,17 @@ object AwsCloudWatchMetricSetup {
       )
     )
 
-  def paymentSuccessRequest(stage: Stage, paymentProvider: PaymentProvider, productType: ProductType): MetricRequest =
+  def paymentSuccessRequest(stage: Stage, isTestUser: Boolean, paymentProvider: PaymentProvider, productType: ProductType): MetricRequest = {
+    val qualifiedStage = stage.toString + (if (isTestUser) "-UAT" else "")
```

Comment:
happy to use something other than UAT.  Also, I considered adding another dimension, but we already have payment method, product, and stage.  Also it would mean updating all the alarms.

But opinions welcome.

Reply:
Why not just UAT?

Reply:
since these are technically coming from the PROD environment I would not want to remove that completely, otherwise we would send the same metric from CODE and PROD if they are a test user.  The touchpoint stage vs code stage is already confusing enough!

Reply:
although having merged it, on reflection we would get hardly any traffic for UAT especially in CODE, and even less likely to be digging into it to debug something.  But anyway it's merged now, happy to +1 a later PR though.